internal/compiler/passes/deduplicate_property_read.rs

```diff
@@ -28,7 +28,7 @@ struct ReadCount {
 
 #[derive(Default)]
 struct PropertyReadCounts {
-    counts: HashMap<NamedReference, ReadCount>,
+    counts: BTreeMap<NamedReference, ReadCount>,
```

Comment:
I don't think this would help in this case since the output shouldn't depend on the order of this. (Unless I'm mistaken which could be) But we only iterate over these to add things in a different BTreeMap)

Would you have an example in which this result in different output?

Reply:
You're right, the original issue is already fixed on your side!

The patch was made for an older branch and then forward ported, and I didn't pay enough attention. I'll be more careful next time.